tests/layer_tests/pytorch_tests/test_unique.py

```diff
@@ -75,5 +77,7 @@ def forward(self, x):
     @pytest.mark.nightly
     @pytest.mark.precommit
     def test_unique2(self, input_shape, sorted, return_inverse, return_counts, ie_device, precision, ir_version):
+        if sys.platform == "win32" and input_shape == [16, 3, 32, 32] and parse_version(torch.__version__).release == (2, 4, 0):
```

Comment:
can you please stay comment why this need to not forget update if new version released

Reply:
It is written in the reason, also with release of 2.4.1 it wouldn't trigger skip anyway